lib/PhasarLLVM/Utils/Annotation.cpp

```diff
@@ -55,7 +56,8 @@ uint64_t VarAnnotation::getLine() const {
 llvm::StringRef VarAnnotation::retrieveString(unsigned Idx) const {
   if (const auto *ConstExpr = llvm::dyn_cast<llvm::ConstantExpr>(
           AnnotationCall->getArgOperand(Idx))) {
-    if (ConstExpr->isGEPWithNoNotionalOverIndexing()) {
+    if (true // ConstExpr->isGEPWithNoNotionalOverIndexing()
```

Comment:
```suggestion
    if (llvm::isa<llvm::GEPOperator>(ConstExpr)
```

Reply:
Is that equivalent? What is the purpose of this if anyhow?

Reply:
No, it is not equivalent, but as close as we can get. 
When you pass a constant string to a function, the string data is a global variable of an array type, so you always pass a constant GEP to that global instead. 

In the ctor of VarAnnotation we don't check that the AnnotationCall is indeed a call to an llvm.var.annotation intrinsic, so in theory this class could be reused for some other purpose. However, in our use-case, this if-check should always succeed